data/mods/More_Survival_Tools/materials.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Also stupid is the fact that tin, lead, and some other items of that sort are instead made of steel. Since I'm not porting over bronze from the Medieval half of this mod, that's beyond the scope of this mod now.